src/controllers/movies/movies.js

```diff
@@ -34,6 +34,7 @@ export default function (view, params, tabContent, options) {
         isLoading = true;
         loading.show();
         const newQuery = { ...query, SortBy: 'Random', StartIndex: 0, Limit: 300 };
+        newQuery.Fields = [...new Set(query.Fields.split(',').concat(['Chapters', 'Trickplay']))].join(',');
```

Comment:
question: should this be added directly in the `query` object instead (ie. at line 281 below)?
this would seem cleaner to me, but I am not sure about the implications ...